### PR TITLE
stdin2syslog: directly exec logger

### PR DESCRIPTION
**Pull request by kpengboy**
Now that we're on stretch and this works

### CODE REVIEW

File: staff/sys/stdin2syslog [hunk 1]
```diff
@@ -1,49 +1,22 @@
...
-def main(argv=None):
-    parser = argparse.ArgumentParser(
-        description=__doc__,
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-    )
-    parser.add_argument('stream')
-    args = parser.parse_args(argv)
+You can use arbitrary stream names (without configuring them before); each
+stream will show up at syslog:/var/log/remote/<stream>.log.
+"
```
Comment (jvperrin) [inline staff/sys/stdin2syslog]: Not a huge fan of the indentation here, but not sure what would be better, maybe a here doc or something?

Comment (matthew-mcallister) [reply staff/sys/stdin2syslog]: Usually you would either save it to a variable at top level indentation or just use multiple `echo` commands.

**Comment by kpengboy** [comment]
I decided to switch the script to bash after trying versions in both languages. I ultimately thought that, for this particular script, the bash version was overall clearer, as it had less syntax getting in the way of communicating the main idea of what the script does.

File: staff/sys/stdin2syslog [hunk 2]
```diff
@@ -1,49 +1,22 @@
...
-import subprocess
-import sys
-
-
-VALID_STREAM_NAME = re.compile('^[a-z0-9\-_]+$')
-
-
-class LoggingFailure(Exception):
-    pass
+if [ $# -lt 1 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
```
Comment (matthew-mcallister) [inline staff/sys/stdin2syslog]: It just occurred to me: you might want to check `$# -ne 1` instead since the script expects exactly one argument.

Comment (kpengboy) [reply staff/sys/stdin2syslog]: Good point.

**Pull request merged**